libraries/chain/include/graphene/chain/withdraw_permission_object.hpp

```diff
@@ -69,6 +110,23 @@ namespace graphene { namespace chain {
               ? withdrawal_limit.amount - claimed_this_period
               : 0, withdrawal_limit.asset_id );
         }
+
+       /***
+        * Get a description of the current withdrawal period
+        * @param current_time   Current time
+        * @return A description of the current period
+        */
+       withdrawal_period_descriptor current_period(fc::time_point_sec current_time) const {
```

Comment:
AFAICS the withdrawal_period_descriptor and this method are only used in unit tests. IMO they shouldn't become part of the main code in that case.